src/elements/filters/FeGaussianBlur.tsx

```diff
@@ -6,12 +6,12 @@ import {
   extractFilter,
   extractIn,
 } from '../../lib/extract/extractFilter';
-import { FilterEdgeMode, NumberProp } from '../../lib/extract/types';
+import { FilterEdgeMode, NumberArray } from '../../lib/extract/types';
```

Comment:
Same here

Reply:
Reverted in [31af51b](https://github.com/software-mansion/react-native-svg/pull/2392/commits/31af51bf062a2054b5082d2693829a2f7b3e46a4)